src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -39,6 +39,10 @@ public void loadSomeData() {
 				} catch (final FileNotFoundException e) {
 				}
 			}
+			
+			if(count > 0) {
+				System.out.println("count > 0");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)